recipes/cad1/task1/baseline/enhance.py

```diff
@@ -372,38 +420,58 @@ def enhance(config: DictConfig) -> None:
         # save processed stems
         n_samples = processed_stems[list(processed_stems.keys())[0]].shape[0]
         out_left, out_right = np.zeros(n_samples), np.zeros(n_samples)
-        for stem_str, item in processed_stems.items():
+        for stem_str, stem_signal in processed_stems.items():
```

Comment:
I think it would be clearer if the loop that does the 'remixing' was kept separate from the loop that writes out the stems. i.e.  so that the code has a clearer decomposition into three stages

- demix
- remix
- save stems and remix

It probably also needs to be clear that the stem outputs and the remixed outputs are in different formats. i.e. the stems are saving a scaling constant (so that they can be reconstructed as float) whereas the mix is being clipped to 16bit.